scripts/create_metadata

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+# OUT_DIR is where the file gets written.
+# The main purpose of this is to make testing easier.
+OUT_DIR="${OUT_DIR:-.}"
```

Comment:
I added this for the benefit of the tests, so we can write test data into different dirs per test to keep it isolated.